Libs/MRML/DisplayableManager/vtkMRMLModelSliceDisplayableManager.cxx

```diff
@@ -605,8 +605,11 @@ ::UpdateDisplayNodePipeline(vtkMRMLDisplayNode* displayNode, const Pipeline* pip
         // that lookup table original range.
         vtkSmartPointer<vtkLookupTable> dNodeLUT = vtkSmartPointer<vtkLookupTable>::Take(displayNode->GetColorNode() ?
           displayNode->GetColorNode()->CreateLookupTableCopy() : nullptr);
-        dNodeLUT->SetAlpha(hierarchyOpacity * displayNode->GetSliceIntersectionOpacity());
-        mapper->SetLookupTable(dNodeLUT);
+        if (dNodeLUT)
+          {
+          dNodeLUT->SetAlpha(hierarchyOpacity* displayNode->GetSliceIntersectionOpacity());
+          mapper->SetLookupTable(dNodeLUT);
```

Comment:
Shouldn't we still call mapper->SetLookupTable if dNodeLUT is nullptr?

Reply:
Yeah, you're right. Changed!